editor/question.js

```diff
@@ -502,4 +502,26 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
     ctrl.init();
   }]);
 
+  angular.module('org.ekstep.question').service('questionServices', ['$http', function ($http) {
+    
+    this.invokeAssetBrowser = function(callbackObj) {
+      ecEditor.dispatchEvent('org.ekstep.assetbrowser:show', callbackObj);
+    }
+  
+    this.generateTelemetry = function(data, plugin){
```

Comment:
Use https://github.com/sunbird-content-plugins/org.ekstep.questionset/blob/master/renderer/utils/telemetry_logger.js

Reply:
The mentioned telemetry is resides inside the question set renderer

Telemetry service here is written for editor events

Is there any way to include Question set utility library independently inside `org.ekstep.question/editor`